about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+        <meta charset="UTF-8">
+        <title>about</title>
+        <script src="js/index.js"></script>
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
+                integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+                crossorigin="anonymous">
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.1/font/bootstrap-icons.css">
+        <link rel="stylesheet" href="css/index.css" />
+        <link href='https://fonts.googleapis.com/css?family=Ubuntu' rel='stylesheet'>
+</head>
+
+<body>
+        <div class="position-relative">
+                <div class="position-absolute top-0 end-0">
+                        <div class="dropdown">
+                                <button type="button" class="btn btn-transparent" data-bs-toggle="dropdown">
+                                        <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30"
+                                                class="bi bi-list" viewBox="0 0 16 16">
+                                                <path fill-rule="evenodd"
+                                                        d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z" />
+                                        </svg>
+                                </button>
+                                <ul class="dropdown-menu">
+                                        <li><a class="dropdown-item" href="index.html">Home</a></li>
+                                        <li><a class="dropdown-item" href="about.html">Über Uns</a></li>
+                                        <li><a class="dropdown-item" href="kontakt.html">Kontakt</a></li>
+                                        <li><a class="dropdown-item" href="impressum.html">Impressum</a></li>
+                                </ul>
+                        </div>
+                </div>
+        </div>
```

Comment:
missing navigation component. Have a look at the Bootstrap dock.
Also the logo and the navigation design in general is missing. Have a look at the created design.

![Bildschirmfoto 2023-11-12 um 21 43 15](https://github.com/nicosammito/IUExam2023/assets/60352747/d6a77e6e-c82d-4792-9d1d-11ba4999a5b8)